### ISSUE TITLE
Immunizations III: Immunization list for person forms [2]

### ISSUE THREAD

**Issue by MateStrysewske** [hzi-braunschweig/SORMAS-Project#5828]
### Situation Description

Concept issue for immunizations: #4081

### Feature Description

Extend the person form (accessed via the person directory) with a list of immunizations associated to that person:

For all the new elements, ONLY show them if the `IMMUNIZATION_MANAGEMENT` feature configuration is enabled.

* Add another card to the right side of the person form below the events; behaves exactly like the other lists
* Only accessible for users with the `IMMUNIZATION_VIEW` user right
* Show the following information for individual list entries:
  * Immunization UUID
  * Disease
  * Means of immunization
  * Management status
  * Immunization status
  * Immunization period (defined by start and end date)
* Allow the user to open the edit form of the immunization by clicking on an edit icon to the right

**Comment by sergiupacurariu** [comment]
this ticket can be merged only after #4764 is merged

**Comment by AndyBakcsy-she** [comment]
Validated ticket on the latest version of Sormas 1.64.0-snapshot on local environment
